### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by wanderingweights**
* Organic repo fetch 
* SQL fixes on get account 
* Var name changes 
* Full network status return

### CODE REVIEW

File: validator/core/config.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Config:
     set_metagraph_weights_with_high_updated_to_not_dereg: bool
     testnet: bool = os.getenv("SUBTENSOR_NETWORK", "").lower() == "test"
     debug: bool = os.getenv("ENV", "prod").lower() != "prod"
+    default_organic_task_history_hours: int = 5
```
Comment (wanderingweights) [inline validator/core/config.py:38]: What does this mean?

Comment (besimray) [reply validator/core/config.py:38]: The get organic tasks endpoint takes an argument of how many hours to look back for. This is the default value. 
Open to suggestions for a better name

Comment (namoray) [reply validator/core/config.py:38]: Why is this is config? This belongs on the endpoint, not in config

File: validator/core/config.py [hunk 2]
```diff
@@ -75,6 +74,8 @@ def load_config() -> Config:
             os.getenv("SET_METAGRAPH_WEIGHTS_WITH_HIGH_UPDATED_TO_NOT_DEREG", "false").lower() == "true"
         )
 
+        default_organic_task_history_hours = int(os.getenv("DEFAULT_ORGANIC_TASK_HISTORY_HOURS", "5"))
```
Comment (namoray) [inline validator/core/config.py:77]: Belongs in the endpoint hardcoded rather than an env var

File: validator/db/sql/tasks.py [hunk 3]
```diff
@@ -337,6 +350,47 @@ async def get_tasks(psql_db: PSQLDB, limit: int = 100, offset: int = 0) -> List[
 
 
 async def get_tasks_by_account_id(psql_db: PSQLDB, account_id: UUID, limit: int = 100, offset: int = 0) -> List[Task]:
```
Comment (namoray) [inline validator/db/sql/tasks.py:352]: Wasn't this endpoint already in?

File: validator/endpoints/tasks.py [hunk 4]
```diff
@@ -203,18 +204,52 @@ async def get_leaderboard(
...
     config: Config = Depends(get_config),
-) -> TrainingTaskStatus:
+) -> NetworkStats:
     try:
         logger.debug("IN get network status")
-        training_stats = await task_sql.get_training_tasks_stats(config.psql_db)
-        if training_stats.number_of_jobs_training >= MAX_CONCURRENT_JOBS:
-            training_stats.job_can_be_made = False
-        return training_stats
+        stats = await task_sql.get_current_task_stats(config.psql_db)
```
Comment (namoray) [inline validator/endpoints/tasks.py:210]: current_task_stats =

File: validator/endpoints/tasks.py [hunk 5]
```diff
@@ -203,18 +204,52 @@ async def get_leaderboard(
...
         logger.info(f"There was an issue with getting training status {e}")
         raise HTTPException(status_code=500, detail=str(e))
 
 
+async def get_completed_organic_tasks(
+    hours: int | None = None,
+    config: Config = Depends(get_config),
+) -> List[TaskDetails]:
+    """Get all completed organic tasks from the last N hours"""
+    lookup_hours = hours if hours is not None else config.default_organic_task_history_hours
```
Comment (namoray) [inline validator/endpoints/tasks.py:224]: should be a default directly in the args, with pydantic field or query so you can explain what the param is directly as well

File: validator/endpoints/tasks.py [hunk 6]
```diff
@@ -203,18 +204,52 @@ async def get_leaderboard(
...
+        return stats
     except Exception as e:
         logger.info(f"There was an issue with getting training status {e}")
         raise HTTPException(status_code=500, detail=str(e))
 
 
+async def get_completed_organic_tasks(
+    hours: int | None = None,
+    config: Config = Depends(get_config),
+) -> List[TaskDetails]:
```
Comment (namoray) [inline validator/endpoints/tasks.py:222]: list not List

File: validator/db/sql/tasks.py (hunk 3 above)
Comment (wanderingweights) [reply validator/db/sql/tasks.py:352]: It was, not sure why git is showing this as completely new - it's a fix

File: validator/core/config.py (hunk 1 above)
Comment (besimray) [reply validator/core/config.py]: good point. Moved to the endpoint

File: validator/endpoints/tasks.py (hunk 6 above)
Comment (besimray) [reply validator/endpoints/tasks.py]: ty. Changed across the file now

File: validator/core/config.py [hunk 7]
```diff
@@ -43,8 +43,6 @@ class Config:
 def load_config() -> Config:
     global _config
     if _config is None:
-
```
Comment (namoray) [inline validator/core/config.py:46]: $10 says we undo this in 2 prs time

**Review by namoray — APPROVED** [review]

**Pull request merged**